src/components/trade/swap/index.tsx

```diff
@@ -118,7 +118,7 @@ const QuickTrade = (props: QuickTradeProps) => {
     const proRatedMarketPrice = tokenFiat * Number(navTokenAmount)
     const proRatedNavPrice = nav * Number(navTokenAmount)
     const navDivergence =
-      (proRatedMarketPrice - proRatedNavPrice) / proRatedMarketPrice / 100
+      (proRatedMarketPrice - proRatedNavPrice) / proRatedNavPrice * 100
```

Comment:
Think this is right...assume market price of 125 and nav price of 100: `(125 - 100) / 100 * 100` => displayed as `25%` below